config/settings.yml

```diff
@@ -1157,3 +1157,8 @@ bid:
 # Settings for Test User Dashboard modules
 test_user_dashboard:
   env: dev
+
+# Settings for mobile module api. Currently unused. They require being added to the devops repo settings to be used.
```

Comment:
Should we delete these settings and add them back when they are actually being used by the API?